modules/terraform-aws-alternat/main.tf

```diff
@@ -185,6 +185,10 @@ data "cloudinit_config" "config" {
     content_type = "text/x-shellscript"
     content      = file("${path.module}/../../scripts/alternat.sh")
   }
+  part {
```

Comment:
Tady fakt jiná cesta není, nakonec mi to přijde jak oka řešení. Jen mě zajímá, co to udělá když to dostane prázdný string?

Reply:
protože [content](https://registry.terraform.io/providers/hashicorp/cloudinit/2.2.0/docs/data-sources/cloudinit_config#content) je required

Reply:
jo tak to nevím, to bych vyzkoušel až jim to budeme reportovat